tta-server/app/models/Population.scala

```diff
@@ -0,0 +1,24 @@
+package models
+
+object Population {
+
+  def generateIncreasePopulationAction(playerState: PlayerState): Action = new Action{
+
+    val costToBuild = 2
+
+    override def doIt(gameState: GameState): GameState = {
+      gameState.updatedActivePlayerState ( playerState =>
```

Comment:
your positioning and spacing for this anonymous function is correct and you just have the wrong type of braces. use curly braces for multiline anonymous functions